apps/web/screens/UniversitySearch/UniversitySearch.tsx

```diff
@@ -912,308 +963,320 @@ const UniversitySearch: Screen<UniversitySearchProps> = ({
-                                  universities.filter(
-                                    (x) => x.id === dataItem.universityId,
-                                  )[0]?.contentfulLogoUrl || ''
+            {loading ? (
+              <>{loadSkeletons()}</>
+            ) : (
+              <>
+                {!gridView && !isMobileScreenWidth && !isTabletScreenWidth && (
+                  <Box>
+                    {filteredResults &&
+                      filteredResults
+                        .slice(
+                          (selectedPage - 1) * ITEMS_PER_PAGE,
+                          (selectedPage - 1) * ITEMS_PER_PAGE + ITEMS_PER_PAGE,
+                        )
+                        .map((item, index) => {
+                          const dataItem =
+                            item.item as UniversityGatewayProgramWithStatus
+                          const specializedName =
+                            locale === 'en'
+                              ? dataItem.specializationNameEn ?? undefined
+                              : dataItem.specializationNameIs ?? undefined
+                          const subHeading =
+                            specializedName !== undefined
+                              ? (locale === 'en'
+                                  ? 'Specialization: '
+                                  : 'Kjörsvið: ') + specializedName
+                              : undefined
+                          return (
+                            <Box marginBottom={3} key={index}>
+                              <ActionCategoryCard
+                                key={`${index}-${dataItem.id}`}
+                                href={
+                                  linkResolver('universitysearchdetails', [
+                                    dataItem.id,
+                                  ]).href
                                 }
-                                alt={`Logo fyrir ${
+                                heading={
                                   locale === 'en'
                                     ? dataItem.nameEn
                                     : dataItem.nameIs
-                                }`}
+                                }
+                                subHeading={subHeading}
+                                text={
+                                  locale === 'en'
+                                    ? stripHtml(dataItem.descriptionEn)
+                                    : stripHtml(dataItem.descriptionIs)
+                                }
+                                icon={
+                                  <img
+                                    src={
+                                      universities.filter(
+                                        (x) => x.id === dataItem.universityId,
+                                      )[0]?.contentfulLogoUrl || ''
+                                    }
+                                    alt={`Logo fyrir ${
```

Comment:
Just a note, perhaps having `${...} logo` instead of `Logo fyrir ${...}` might be better

Reply:
Good point, Ill update this